src/templates/apis.j2

```diff
@@ -52,7 +52,11 @@ class {{ safe_class_name(class_name) }}:
                 raise ValidationError(ModelApiResult.from_dict(response.json()).to_dict())
 {% elif response_code["code"] in (200, 201) %}
                 self.logger.info("{{ response_code["message"] }}")
-            return {{ operation.get_response_str(response_code["code"]) }}
+            if type(response.json()) is list:
```

Comment:
Just a note, the recommended way to compare types is to use isinstance, it's marginally faster and there are some inheritance edge cases isinstance will cover, a review of this can be found in -> https://blog.devgenius.io/what-is-better-type-vs-isinstance-3b948d1b81b5

Reply:
This is great, thanks for sharing Michael. We actually use type() a fair bit in our codebase, might be worthwhile to revisit. For now I updated this PR. Thanks Michael.